Userland/Libraries/LibGfx/Point.h

```diff
@@ -214,6 +214,12 @@ class Point {
         return y() - other.y();
     }
 
+    [[nodiscard]] Point<T> clamp_to(Rect<T> const& input) const
+    {
+        return Point<T>(AK::clamp<T>(x(), input.x(), input.x() + input.width()),
+            AK::clamp<T>(y(), input.y(), input.y() + input.height()));
+    }
```

Comment:
Is this different from `Point::constrain(Rect)`?

Reply:
Indeed it is not. Welp